src/plugins/kibana_overview/public/components/overview/overview.test.tsx

```diff
@@ -148,27 +124,94 @@ const mockFeatures = [
   },
 ];
 
+const flushPromises = async () => {
+  await new Promise<void>(async (resolve, reject) => {
+    try {
+      setTimeout(() => resolve(), 10);
+    } catch (error) {
+      reject(error);
+    }
+  });
+};
+
+const updateComponent = async (component: ReactWrapper) => {
+  await act(async () => {
+    await flushPromises();
+    component.update();
+  });
+};
+
 describe('Overview', () => {
-  test('render', () => {
-    const component = shallowWithIntl(
+  beforeEach(() => {
+    hasUserDataViewMock.mockClear();
+    hasUserDataViewMock.mockResolvedValue(true);
+  });
+
+  afterAll(() => jest.clearAllMocks());
+
+  test('render', async () => {
+    const component = mountWithIntl(
```

Comment:
I had to switch from `shallow` to `mount` because enzyme does not support `useEffect` with shallow rendering, so all snapshots were in the `loading` state this PR adds. The snapshots are ugly, but correct.